From 64160738b9bd9dfd120381268b9e242894ef6c75 Mon Sep 17 00:00:00 2001
From: sittuk-kibet <kibetnathan@kabarak.ac.ke>
Date: Sat, 7 Jun 2025 12:57:43 +0300
Subject: [PATCH] YAML file  would create a Pod with a single container with a
 volume named scratch-volume created, which would create the /scratch
 directory inside the container.

---
 volumeSpec.yaml | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 volumeSpec.yaml

diff --git a/volumeSpec.yaml b/volumeSpec.yaml
new file mode 100644
index 0000000..5274a73
--- /dev/null
+++ b/volumeSpec.yaml
@@ -0,0 +1,18 @@
+---
+apiversion: v1
+kind: pod
+metadata:
+  name: busybox
+  namespace: default
+spec: 
+containers:
+  - image: busybox
+    name: busy
+command:
+  - sleep
+  - "3600"
+volumeMounts:
+  - mountPath: /scratch
+    name: scratch-volume
+volumes:
+  - name: scratch-volume emptyDir{}
-- 
2.48.1

